javascript-methods/main.js

```diff
@@ -0,0 +1,61 @@
+var a = 123;
+var b = 456;
+var c = 789;
+
+var maxValue = Math.max(a, b, c);
+console.log('maxValue:', maxValue);
+
+var heroes = [
+  'Iskandar, King of Conquerors',
+  'Sasaki Kojiro',
+  'Diarmuid Ua Duibhne',
+  'Gilles de Rais'
+];
+
+var randomNumber = Math.random();
+randomNumber = randomNumber * heroes.length;
+
+var randomIndex = Math.floor(randomNumber);
+console.log('randomIndex:', randomIndex);
+
+var randomHero = heroes[randomIndex];
+console.log('randomHero:', randomHero);
+
+var library = [
+  {
+    title: 'I Want To Eat Your Pancreas',
+    author: 'Yoru Sumino'
+  },
+  {
+    title: 'Spy X Family',
+    author: 'Tatsuya Endo'
+  }
+];
```

Comment:
Your initial library should have three books:

---

![image](https://user-images.githubusercontent.com/16143771/177655307-72b735fa-9596-4505-ab8c-1344d7d23928.png)

---

At the end of the Array Methods section, the library should still have two books left after making the above changes:

---

![image](https://user-images.githubusercontent.com/16143771/177655487-dac160f2-c046-4182-b3d2-1cf416c12ace.png)